packages/api-plugin-fulfillment/src/resolvers/Query/getFulfillmentTypes.js

```diff
@@ -0,0 +1,31 @@
+import getPaginatedResponse from "@reactioncommerce/api-utils/graphql/getPaginatedResponse.js";
+import wasFieldRequested from "@reactioncommerce/api-utils/graphql/wasFieldRequested.js";
+import { decodeShopOpaqueId } from "../../xforms/id.js";
+
+/**
+ * @name Query/FulfillmenTypes
+ * @method
+ * @memberof Fulfillment/Query
+ * @summary Query for a list of fulfillment types
+ * @param {Object} _ - unused
+ * @param {Object} args - an object of all arguments that were sent by the client
+ * @param {String} args.shopId - Shop ID to get records for
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} info Info about the GraphQL request
+ * @returns {Promise<Object>} Fulfillment methods
```

Comment:
Is this correct? If we are just returning one object why are we using pagination?

Reply:
We are returning all the fulfillment types under the shop.
Fixed the 'return' text.